### ISSUE TITLE
Add a command to disable a rule for a single line

### ISSUE THREAD

**Issue by jpsim** [realm/SwiftLint#222]
For example:

```swift
// swiftlint:disable:next force_cast
let forceCastNoWarning = NSNumber() as! Int
let forceCastWithWarning = NSNumber() as! Int
```

Although I'm not convinced on the naming. Maybe `// swiftlint:disable:X` where `X` is an integer number of lines for which to disable the rule?

It would make it more convenient to temporarily disable rules for occasional violations.

**Comment by segiddins** [comment]
Why not just allow putting it at the end of the line, like rubocop does?

**Comment by jpsim** [comment]
> Why not just allow putting it at the end of the line, like rubocop does?

so `// swiftlint:disable rule_name` would start a region if preceded by whitespace, but start and close a single-line region otherwise?

**Comment by segiddins** [comment]
Yeah. Not necessarily saying it's better, and it might clash with the line length rule

**Comment by richellis** [comment]
+1 for the suggestion about putting it at the end of the line. Perhaps an abbreviated syntax to mitigate line length issues, such as `//sl: rule_name`

**Comment by mpurland** [comment]
+1 for this suggestion

**Comment by jpsim** [comment]
Done in #292.
